user_list.php

```diff
@@ -29,64 +29,46 @@
 			<?php
 			  require("nav.php");
 			?>
 			
-                                <div class="aside"><!-- holds the admin menu with links to the different pages only the admin can access -->
-									<ul>
-										<li><a href="user_list.php">User List</a></li> 
-										<li><a href="add_user.php">Add New User</a></li>
-										<li><a href="update_user.php">Update User</a></li>
-										<li><a href="delete_user.php">Delete User</a></li>
-									</ul>
-								</div>
-			
-			
-                                <div class="admin_content"><!-- Holds the main page content -->
-								<h2>USERS</h2>
-								<?php
-									//connect.php (tells where to connect servername, username, password, dbaseName)
-									require "3.3_Assessment_FTuason_mysqli.php";
+                                <div class="content" style="padding-top: 0;"><!-- holds the main page content -->
+								<heading3><!-- This class is to split the content in half and places the usernames to the left-->
+									<User1><h4>USERNAME</h4></User1>
+									<Password1><h4>PASSWORD</h4></Password1>
+								</heading3>
+									
+										<?php
+										//connect.php (tells where to connect servername, username, password, dbaseName)
+										require "3.3_Assessment_FTuason_mysqli.php";
 
-									//creates a variable to store sql code for the 'display all users' query
-									$query = ("SELECT * FROM Users");
+										//creates a variable to store sql code for the 'display all users' query
+										$query = ("SELECT * FROM Users");
 
-									//runs the query to display all the users usernames
-									$result = mysqli_query($conn,$query);
-
-									while ($output = mysqli_fetch_array($result))
-									{
-										echo "<user2>" . $output['User_PK'] . "</user2><br>";
-									//closes the while loop
-										}
-								?>
-								<h2>PASSWORDS</h2>
-								<?php
-									//runs the query to display all the users passwords
-									$result = mysqli_query($conn,$query);
+										//runs the query to display all the users usernames
+										$result = mysqli_query($conn,$query);
 
-									while ($output = mysqli_fetch_array($result))
-									{
-										echo "<user2>" . $output['Password'] . "</user2><br>";
-									//closes the while loop
+										while ($output = mysqli_fetch_array($result))
+										{
+										?>
+										<heading4><!-- This class is to split the content in half and places the passwords to the right-->
+											<User2 style="padding-left: 5px;"><p><?php echo $output['User_PK']; ?></p></User2>
+											<Password2><p><?php echo $output['Password']; ?></p></Password2>
+										</heading4>
+										<?php
+										//closes the output while loop
 										}
-								?>
+										?>
                         		</div>
 			
                      <!-- This class is for my footer-->	
 					<div class="footer">
 						<h1>Footer</h1>
-						<button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>
```

Comment:
#back to top button that would bring the user to the top of the page once they reached a certain point on the page